rbac/management/group/view.py

```diff
@@ -556,7 +571,14 @@ def obtain_roles(self, request, group):
 
         filtered_roles = self.filtered_roles(roles, request)
 
-        return [RoleMinimumSerializer(role).data for role in filtered_roles]
+        annotated_roles = filtered_roles.annotate(policyCount=Count('policies', distinct=True))
+
+        if ORDERING_PARAM in request.query_params:
+            ordered_roles = self.order_queryset(annotated_roles, VALID_ROLE_ORDER_FIELDS,
+                                                request.query_params.get(ORDERING_PARAM))
+            if ordered_roles:
```

Comment:
Since you have added validation error in the order_queryset method, maybe we could get rid of this if statement?

Reply:
Good point!